### PR TITLE
Increase CP version check accuracy, bump dep for clickstream to 5.0.1

### PR DESCRIPTION
**Pull request by rmoff**
@ybyzek I've tested this but would appreciate a second set of eyes on it please :)

### CODE REVIEW

**Comment by rmoff** [comment]
hmm some other commits bundled in, let me remove those

File: utils/helper.sh [hunk 1]
```diff
@@ -31,7 +31,8 @@ function check_ccloud() {
 function check_running_cp() {
   expected_version=$1
 
-  actual_version=$( confluent version | tail -1 | awk -F':' '{print $2;}' | awk '$1 > 0 { print substr($1,1,3)}' )
+  # This returns the actual version running trimmed to the same accuracy as the supplied expected_version
+  actual_version=$( confluent version | tail -1 | awk -F': ' -v exp_ver=$expected_version '{print substr($2,1,length(exp_ver))}')
```
Comment (ybyzek) [inline utils/helper.sh:35]: Clever enhancement @rmoff !

File: clickstream/start.sh [hunk 2]
```diff
@@ -4,9 +4,9 @@
 . ../utils/helper.sh
 
 check_env || exit 1
-check_running_cp 5.0 || exit 
-check_running_elasticsearch 5.6.5 || exit 1
-check_running_grafana 5.0.3 || exit 1
+check_running_cp 5.0.1 || exit 
+#check_running_elasticsearch 5.6.5 || exit 1
```
Comment (ybyzek) [inline clickstream/start.sh]: Why is `check_running_elasticsearch` and `check_running_grafana` commented out?

File: clickstream/start.sh [hunk 3]
```diff
@@ -4,9 +4,9 @@
 . ../utils/helper.sh
 
 check_env || exit 1
-check_running_cp 5.0 || exit 
-check_running_elasticsearch 5.6.5 || exit 1
-check_running_grafana 5.0.3 || exit 1
+check_running_cp 5.0.1 || exit
```
Comment (ybyzek) [inline clickstream/start.sh:7]: Is there a reason CP *must* be running 5.0.1?  Will it fail with 5.0.0 or 5.0.2?  If it works with 5.0.0 and 5.0.2, then why not leave as 5.0?

**Review by ybyzek — CHANGES_REQUESTED** [review]
Some questions to discuss before merging.

File: clickstream/start.sh (hunk 2 above)
Comment (rmoff) [reply clickstream/start.sh]: by mistake, reverted in https://github.com/confluentinc/examples/pull/85/commits/80292d5a7ae549f9eca39f7b363aaacb2e6da455

File: clickstream/start.sh (hunk 3 above)
Comment (rmoff) [reply clickstream/start.sh:7]: there's a bug in RUN SCRIPT which was fixed in 5.0.1, so yes it'll fail in 5.0.0. 
The dep is basically >=5.0.1

**Comment by ybyzek** [comment]
PR is out of date and has been handled in recent versions.

**Pull request closed without merging**